source/endpoints/devices/Activator.py

```diff
@@ -0,0 +1,41 @@
+from flask_restplus import Resource
+from flask_restplus import fields
+
+from endpoints.devices import ns, DAO
+from endpoints.util.StateTypeToString import StateTypeToString
+from endpoints.util.ToString import ToString
+
+activator = ns.model('Activator', {
+    'activatorId': fields.Integer(readOnly=True, required=True, discription='The unique identifier of the action')
+    ,'name': fields.String(readOnly=True, required=True, description='The name of the action')
+    ,'stateType': StateTypeToString(attribute="stateType", readOnly=True, required=True, description='The type of interaction you can have')
+    ,'stateString': ToString(attribute="state", required=True, discription='The state of an action')
+    , "requiredInfo": ToString(attribgute="requiredInfo", required=True, discription="The info needed to operate")
+})
+state = ns.model( 'State', {
+    'state': fields.String(required = True, discription = 'new state of activator')
+})
+
+@ns.route('/<int:deviceId>/activator/<int:activatorId>')
+@ns.response(404, 'Device or action not found')
+@ns.param('deviceId', 'The device ID')
+@ns.param('activatorId', 'The action ID')
+class Activator(Resource):
+    ''' Show a single action, and put a new value'''
+    @ns.doc('get_activator')
+    @ns.marshal_with(activator)
+    def get(self, deviceId, activatorId):
+        ''' Fetch a given action of a device '''
+        device = DAO.getDevice(deviceId)
```

Comment:
Since we call it directly, maybe call the DAO class just Database to make it more clear